### DISCUSSION TITLE
SDRplay RSPdx Antenna C port?

### DISCUSSION THREAD

**Discussion by DE3JGA** [szpajder/dumphfdl#33] [Q&A]
Hello, 

how can I activate antenna output C (BNC connector) on the SDRplay RSPdx. When I type the command --antenna c it always shows Antenna A and not C. 

Thanks for the help and 73 Josef

**Comment by szpajder** [comment]
If the port name is "C", then you should probably type --antenna C rather
than c, right?

**Reply by DE3JGA** [reply]
Hello Tomasz,

Thank you for your quick response. It doesn't matter if I put "C" or "c". Only antenna output A is activated, never C.

73 and Merry Christmas
Josef

**Comment by szpajder** [comment]
As far as I can see in the driver code, the names of antenna ports of this
radio are: "Antenna A", "Antenna B" and "Antenna C", not just "A", "B" and
"C".

https://github.com/pothosware/SoapySDRPlay3/blob/794425ff7d8cc153190d6545332df604731bab41/Settings.cpp#L172

You must specify these names exactly as advertised by the driver. Use
"SoapySDRUtils --probe=driver=sdrplay" to list supported settings of the
device, including antenna port names.

**Reply by DE3JGA** [reply]
Hello, thanks for the help. My mistake was that I didn't specify the " ".
Thanks again and 73
Josef
